src/StrategyTests.sol

```diff
@@ -88,4 +90,62 @@ contract StrategyTests is OrderBookStrategyTest {
             assertEq(strategyAmount, strategy.expectedAmount);
         }
     }
+
+    function getNamespace(address orderOwner, address contractAddress) internal pure returns (FullyQualifiedNamespace) {
+        return LibNamespace.qualifyNamespace(
+            StateNamespace.wrap(uint256(uint160(orderOwner))),
+            contractAddress
+        );
+    }
+
+    function evalExpression(
+        LibStrategyDeployment.StrategyDeployment memory strategy,
+        FullyQualifiedNamespace namespace,
+        uint256[][] memory context,
+        uint256[] memory inputs,
+        uint256 sourceIndex
+
+    ) internal {
+        bytes memory bytecode = PARSER.parse2(
```

Comment:
shouldn't this be something like `iParser` rather than `PARSER` if it is immutable?